.github/workflows/release.yaml

```diff
@@ -90,16 +90,23 @@ jobs:
           username: ${{ env.USERNAME }}
           password: ${{ secrets.GITHUB_TOKEN }}
       - uses: sigstore/cosign-installer@v3.1.2
+      - name: Obtain ghcr image digest
+        id: ghcr-digest
+        run: |
+          # Pull the ghcr image to get the digest
+          docker pull ${{ matrix.images.image }}
```

Comment:
You don't need to pull to get the digest of an image which was pushed in the same run. Digest is simply a hash assigned by registry on publish.